src/physics/em/detail/PhysicsConstants.hh

```diff
@@ -59,6 +59,22 @@ CELER_CONSTEXPR_FUNCTION MevPerCm lpm_constant()
 }
 //!@}
 
+//!@{
+//! Constant functions for model limits
+
+//! Maximum energy for the SeltzerBerger model to be valid
+CELER_CONSTEXPR_FUNCTION units::MevEnergy seltzer_berger_limit()
+{
+    return units::MevEnergy{1e3}; //! 1 GeV
+}
```

Comment:
I'm ok with the hardcoded relativistic upper energy "constant", but isn't the SB limit based on the input data? If it's not trivial to pull that out we can leave this with a `TODO` marker.

Reply:
You are correct that the upper limit of the SB model is a configurable physics parameter.  This seltzer_berger_limit is a common default value for the upper limit of the SB model and the low limit of the RB model, which was a part shared data and factored out to here for a consistency.  Alternatively, we may move this to BremsstrahlungProcess::Options as a configurable input value later.